src/SFA.DAS.AssessorService.Application.Api.Client/Clients/ApplicationApiClient.cs

```diff
@@ -7,6 +7,8 @@
 using SFA.DAS.AssessorService.Api.Types.Models.AO;
 using SFA.DAS.AssessorService.Api.Types.Models.Apply;
 using SFA.DAS.QnA.Api.Types.Page;
+using SFA.DAS.QnA.Api.Types;
+using SFA.DAS.AssessorService.ApplyTypes;
```

Comment:
Would there be ApplyTypes as I thought these were the types in the Apply API which is being removed?

Reply:
Still need them until fully migrated.